i18n/oceans.json

```diff
@@ -0,0 +1,107 @@
+    "fishvtrash-predicting-init3": "Let’s go!",
+    "fishvtrash-pond-init1": "Based on your training, here are some objects that A.I. identified as “fish”.  How did A.I. do?",
+    "fishvtrash-pond-recall": "Click here to switch between objects identified as “fish” and “not fish”.",
+    "fishvtrash-pond-init2": "You can train A.I. more...",
+    "fishvtrash-pond-init3": "...or Continue.",
+    "creaturesvtrashdemo-predicting-init1": "So far, we trained A.I. to identify objects as either “fish” or “not fish”.",
+    "creaturesvtrashdemo-predicting-init2": "What if this training data was used to decide what belonged in the water?",
+    "creaturesvtrashdemo-predicting-init3": "What would happen to other sea creatures?  Does our training approach cause unintended consequences?",
+    "creaturesvtrashdemo-predicting-init4": "Let's see.",
+    "creaturesvtrashdemo-predicting-pause1": "Did you notice that A.I. identified some sea creatures as “not fish”?",
+    "creaturesvtrashdemo-predicting-pause2": "While these sea creatures are not fish, they do belong in the water.",
+    "creaturesvtrashdemo-predicting-pause3": "A.I. only learns what we teach it.",
+    "creaturesvtrashdemo-predicting-pause4": "Let’s train A.I. again.",
+    "creaturesvtrash-training-init1": "Let’s teach A.I. to learn which objects should be in the water.",
+    "creaturesvtrash-training-init2": "In the ocean, plastic debris can harm fish, seabirds, and marine mammals. Protecting seabirds and marine life is one of many reasons to keep the oceans clean.",
+    "creaturesvtrash-training-init3": "The more training data you provide, the more A.I. learns.  Keep training.",
+    "creaturesvtrash-training-init4": "Marine plastic pollution has impacted at least 267 species worldwide, including 86% of all sea turtle species.",
+    "creaturesvtrash-training-init5": "Keep training to teach A.I. what belongs in the water.",
+    "creaturesvtrash-training-init6": "Great work!  The more data you provide A.I., the better it will do.  Keep training A.I. or continue when ready.",
+    "creaturesvtrash-predicting-init1": "Do you think A.I. will now do a better job identifying what should be in the water?  Let’s watch.",
+    "creaturesvtrash-pond-init1": "Based on your training, here are some objects that A.I. identified as “belongs in water”.  How did A.I. do?",
+    "fishshort-words-init1": "Artificial intelligence and machine learning can be used to teach a computer new patterns.",
+    "fishshort-words-init2": "Let’s teach A.I. a new word by showing it examples of that type of fish.",
+    "fishshort-words-training-pause1": "Do you think A.I. has enough training data?  You can click Continue to find out.",
+    "fishshort-words-training-pause2": "Great work!  Keep training A.I. or continue when ready.",
+    "fishshort-predicting-init1": "With your training data, you have programmed A.I. to recognize “{WORD}” fish.",
```

Comment:
do the `WORD` and `N` keys need to be all caps? i think our/js convention is camelcase for i18n args

Reply:
Don't think so. I was following some documentation but if our convention is camel-case I'll change it.